batch/snippets/pom.xml

```diff
@@ -1,13 +1,12 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.google.cloud</groupId>
   <artifactId>-snippets</artifactId>
   <packaging>jar</packaging>
   <name>Google Google Cloud Batch Snippets</name>
-  <url>https://github.com/googleapis/java-batch</url>
```

Comment:
Should we link this to: https://github.com/GoogleCloudPlatform/java-docs-samples/tree/main/batch